src/igvfd/schemas/sequence_data.json

```diff
@@ -0,0 +1,132 @@
+        "uuid",
+        "accession",
+        "alternate_accessions",
+        "aliases",
+        "md5sum"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "file.json#/properties"
+        },
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "file.json#/dependentSchemas"
+        ],
+        "file_format": {
+            "comment": "Fastq files must specify their minimum, maximum, and mean read lengths.",
+            "if": {
+                "properties": {
+                    "file_format": {
+                        "const": "fastq"
+                    }
+                }
+            },
+            "then": {
+                "required": [
+                    "minimum_read_length",
+                    "maximum_read_length",
+                    "mean_read_length"
+                ]
+            }
+        }
+    },
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "content_type": {
+            "comment": "Content Type describes the content of the file. Reads are individual sequences of bases corresponding to DNA or RNA fragments in a FASTQ text file format. Subreads are sequences of bases produced using PacBio platforms.",
+            "enum": [
+                "reads",
+                "subreads"
+            ]
+        },
+        "file_format": {
+            "enum": [
+                "bam",
+                "fastq"
+            ]
+        },
+        "dbxrefs": {
+            "items": {
+                "title": "External identifier",
+                "description": "Identifier from an external resource that may have 1-to-1 or 1-to-many relationships with IGVF file objects.",
+                "type": "string",
+                "pattern": "^(SRA:(SRR|SRX)\\d+)$"
+            }
+        },
+        "platform": {
+            "title": "Platform",
+            "description": "The sequencing platform used to produce the sequence data.",
+            "type": "string"
+        },
+        "minimum_read_length": {
+            "title": "Minimum Read Length",
+            "description": "For high-throughput sequencing, the minimum number of contiguous nucleotides determined by sequencing.",
+            "type": "integer",
+            "minimum": 0
+        },
+        "maximum_read_length": {
+            "title": "Read Length",
+            "description": "For high-throughput sequencing, the maximum number of contiguous nucleotides determined by sequencing.",
+            "type": "integer",
+            "minimum": 0
+        },
+        "mean_read_length": {
+            "title": "Read Length",
```

Comment:
I think read length, as well as reads count, should be calculated and fille din by the validator. So they should be required - after the uploading status changed to validated. not sure it is easy to do in dependencies